include/sys/zio.h

```diff
@@ -161,7 +161,7 @@ enum zio_encrypt {
 /*
  * Default Linux timeout for a sd device.
  */
-#define	ZIO_DELAY_MAX			(30 * MILLISEC)
+#define	ZIO_SLOW_IO_MAX			(30 * MILLISEC)
```

Comment:
Why do we use a macro for this, rather than initializing  `int zio_slow_io_ms = 30 * MILLISEC;`?  I'm not seeing the value in this additional indirection.

I think wherever `30 * MILLISEC` appears, we'll want a comment saying that it's 30 seconds.  (as opposed to assuming that `30 * MILLISEC` means 30 milliseconds).